datacube/api/core.py

```diff
@@ -489,7 +493,8 @@ def load(
             For example, loaded data could be filtered to January observations only by passing the following
             predicate function that returns True for datasets acquired in January::
 
-                def filter_jan(dataset): return dataset.time.begin.month == 1
+                def filter_jan(dataset):
+                    return dataset.time.begin.month == 1
```

Comment:
It's not from this PR, but the single period on line 499 looks quite strange in the rendered documentation.